src/components/FooterPreLogin/FooterPreLogin.tsx

```diff
@@ -41,11 +41,18 @@ export const FooterPreLogin = ({
   ...langProps
 }: FooterPreLoginProps): JSX.Element => {
   const wrapHandleClick =
-    (href: string, linkType: "internal" | "external") =>
+    (
+      linkType: "internal" | "external",
+      href?: string,
+      onClickLink?: () => void
+    ) =>
     (e: React.SyntheticEvent) => {
       if (onExit) {
         e.preventDefault();
-        onExit(href, linkType);
+        onExit(linkType, href);
+      }
+      if (onClickLink) {
+        onClickLink();
```

Comment:
As I see, the onExit function is supposed to take in charge the action to perform: thus the ASIS code is accepting the href... as the same way, you should handle the onClick together with the onExit and call onClick, if defined, only if onExit is not defined